docs/macOS.md

```diff
@@ -23,10 +23,8 @@ sudo make install
 Finally, from the root directory of this repository:
 
 ``` shell
-mkdir build
-cd build
-cmake .. -DCMAKE_TOOLCHAIN_FILE=../darwin.toolchain
-make
+cmake . -B build -DCMAKE_TOOLCHAIN_FILE=./darwin.toolchain
```

Comment:
I'm not sure that a toolchain is actually needed here (at least, I've never needed one before on a mac)